scripts/azureml-assets/azureml/assets/environment/build.py

```diff
@@ -63,6 +65,18 @@ def create_acr_task(image_name: str,
             'cmd': f"$Registry/{image_name} {test_command}"
         })
 
+        if trivy_url is not None:
+            task['steps'].append({
+                'id': 'scan',
+                'cmd': (
+                        f"$Registry/{image_name} "
+                        f"wget -O trivy.deb {trivy_url} && "

```

Comment:
This minimizes output:
```suggestion
                        f"curl -s -S -f -o trivy.deb {trivy_url} && "
```